src/service-object.ts

```diff
@@ -102,7 +102,8 @@ export class GrpcServiceObject extends ServiceObject {
    *
    * @private
    */
-  request(...args: Array<{}>) {
+  // tslint:disable-next-line no-any
+  request(...args: Array<{}>): Promise<[any, r.Response]> {
```

Comment:
Try combining `[any, r.Response]` into a type, something like `RequestResponse = [any, r.Response]`

Reply:
Follow-up: I'm seeing that most of the library is written in this style so my question is, why? Does wrapping things in types lower performance?

Reply:
done